pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -355,3 +359,458 @@ func TestSelectCandidate(t *testing.T) {
 		})
 	}
 }
+
+type fakeCandidate struct {
+	victims *extenderv1.Victims
+	name    string
+}
+
+// Victims returns s.victims.
+func (s *fakeCandidate) Victims() *extenderv1.Victims {
+	return s.victims
+}
+
+// Name returns s.name.
+func (s *fakeCandidate) Name() string {
+	return s.name
+}
+
+func TestPrepareCandidate(t *testing.T) {
```

Comment:
Could you add a one more test case with two victims, where one will pass successfully and second will fail?

Reply:
sure. added